packages/two_dimensional_scrollables/example/lib/tree_view/simple_tree.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:two_dimensional_scrollables/two_dimensional_scrollables.dart';
+
+/// The class containing a TreeView that highlights the selected row. The
+/// default TreeView.treeNodeBuilder, makes tapping the leading icon of a parent
+/// toggle the node open and closed. The scrollbars will appear as the content
+/// exceeds the bounds of the viewport.
+class TreeExample extends StatefulWidget {
```

Comment:
Cannot "click to select" in the non-label area of the row. Only possible to select when clicking in the label area.

https://github.com/flutter/packages/assets/48603081/92976c33-0921-435b-9ef5-03a5ad80c2bc

Reply:
Same in the custom demo.

https://github.com/flutter/packages/assets/48603081/712abd22-9c4c-4eba-b740-f4b1f424756d

Reply:
For instance, VS Code allows selecting the node outside the label area.



https://github.com/flutter/packages/assets/48603081/6eb2039e-3cc6-4a4e-afb7-7dece376e291

Reply:
Fixed in RenderTreeViewport.hitTestChildren, thanks!